Doc/glossary.rst

```diff
@@ -1033,17 +1037,40 @@ Glossary
       :attr:`~instance.__class__` attribute or can be retrieved with
       ``type(obj)``.
 
+   type alias
+      A synonym for a type, created by assigning the type to an identifier.
+
+      Type aliases are useful for simplifying :term:`type hints <type hint>`.
+      For example ::
+
+         from typing import List, Tuple
+
+         Color = Tuple[int, int, int]
+
+         def remove_gray_shades(colors: List[Color]) -> List[Color]:
+             pass
+
+      is equivalent to ::
+
+         def remove_gray_shades(
+                 colors: List[Tuple[int, int, int]]) -> List[Tuple[int, int, int]]:
+             pass
+
+      See :mod:`typing` and :pep:`484`, which describe this functionality.
+
    type hint
-      A specification about the expected type for a global variable, class
-      variable, function or method parameter or return value.
+      An :term:`annotation` that specifies the expected type for a variable, class
+      attribute, function or method parameter or return value.
```

Comment:
Rewrite as "[...] for a variable, a class attribute, or a function parameter or return value." (The repetition of "a" makes it clear that the last "or" binds more tightly than the other.)